runtime/onert/core/src/compiler/ExecutorFactory.h

```diff
@@ -21,14 +21,7 @@
 
 #include "backend/ITensor.h"
 
-#ifdef ONERT_TRAIN
-#include "backend/train/TrainableBackendContext.h"
-#endif // ONERT_TRAIN
 #include "compiler/LoweredGraph.h"
-#ifdef ONERT_TRAIN
-#include "compiler/train/LoweredTrainableGraph.h"
-#include "compiler/train/TrainingInfo.h"
-#endif // ONERT_TRAIN
```

Comment:
#ifdef ... and include train related-headers are replaced with forward declaration below.